core/src/main/java/org/jruby/RubyArray.java

```diff
@@ -4639,6 +4640,26 @@ public RubyString pack(ThreadContext context, IRubyObject obj) {
         }
     }
 
+    @JRubyMethod(name = "pack")
+    public RubyString pack(ThreadContext context, IRubyObject obj, IRubyObject maybeOpts) {
+        Ruby runtime = context.runtime;
+        IRubyObject opts = ArgsUtil.getOptionsArg(runtime, maybeOpts);
+        RubyString str = RubyString.newEmptyString(runtime);
```

Comment:
preferably use `RubyString.newString()` the newEmptyString is for cases such as `return ''`